blueocean-rest/pom.xml

```diff
@@ -100,6 +101,8 @@
                                         <allowedImport>hudson.Extension</allowedImport>
                                         <allowedImport>hudson.ExtensionPoint</allowedImport>
                                         <allowedImport>hudson.ExtensionList</allowedImport>
+                                        <allowedImport>hudson.model.User</allowedImport>
+                                        <allowedImport>hudson.model.UsageStatistics</allowedImport>
```

Comment:
huh - never seen this type of allowed import thing before

Reply:
@vivek and I introduced it to make sure we only put intentional dependencies on core within `blueocean-rest`